src/util/transition/ngbTransition.ts

```diff
@@ -28,6 +28,7 @@ export const ngbRunTransition =
 
           // If animations are disabled, we have to emit a value and complete the observable
           if (!options.animation) {
+            (startFn(element, null !) || noopFn)();
```

Comment:
Why is this needed? it's not clear to me. Is it because even without animations we need to remove the `hidden` class? If yes, let's not put it in case animation is false.

Reply:
I think that's the explanation → https://github.com/ng-bootstrap/ng-bootstrap/pull/3746#issuecomment-635186699

TLDR:
- `[class.fade]: 'animation'` always bindings
- `.show` always added/removed via the `ngbRunTransition`

Othwise you have to "duplicate" adding show class via binding and via `ngbRunTransition`